include/proto/functions.proto

```diff
@@ -80,7 +80,10 @@ message ArgList {
 message DagCall {
   required string name = 1;
   map<string, ArgList> function_args = 2;
-  optional string response_address = 3;
+  required ConsistencyType consistency = 3 [default = NORMAL];
+  optional string response_address = 4;
+  optional string output_key = 5;
+  optional string client_id = 6;
```

Comment:
What is the `client_id`?

Reply:
client id is the session id of the client who is calling the DAG.
This id is used as the vector clock's key.